packages/ninjamsrv/PKGBUILD

```diff
@@ -0,0 +1,44 @@
+# Maintainer: OSAMC <https://github.com/osam-cologne/archlinux-proaudio>
+# Contributor: jakob <grandchild@gmx.net>
+# Contributor: Florian Hülsmann <fh@cbix.de>
+
+_name=ninjam
+pkgname=ninjamsrv
+pkgver=0.080
+pkgrel=1
+pkgdesc='Ninjam online jamming server'
+arch=(x86_64 aarch64)
+url='https://cockos.com/ninjam/'
+license=(GPL2)
+depends=(gcc-libs)
+makedepends=()
+optdepends=()
```

Comment:
Can be removed?

Or should `systemd` be added? (I have no clue what other packages providing systemd service files do it.)

Reply:
Some packages do (like [php7-fpm](https://archlinux.org/packages/extra/x86_64/php7-fpm/)) and others (like [gpsd](https://archlinux.org/packages/community/x86_64/gpsd/)) don't 🤔

I just removed the empty fields on this branch.